sdk/storage/azblob/internal/shared/log_events.go

```diff
@@ -1,7 +1,10 @@
+//go:build go1.18
+// +build go1.18
+
 // Copyright (c) Microsoft Corporation. All rights reserved.
-// Licensed under the MIT License.
+// Licensed under the MIT License. See License.txt in the project root for license information.
 
-package exported
+package shared
```

Comment:
This should go back to the `exported` package after the batching building content has been moved.

Reply:
Done